cli/viam/main.go

```diff
@@ -803,6 +803,38 @@ func main() {
 					},
 				},
 			},
+			{
+				Name:  "module",
+				Usage: "manage your hosted modules",
+				Subcommands: []*cli.Command{
+					{
+						Name:        "create",
+						Usage:       "create & register a module on app.viam.com",
+						Description: `Creates a module in app.viam.com using a public_namespace and a module name.
+Ex: 'viam module create --name my-great-module --public_namespace my-registered-namespace'
+Will create the 'my-registered-namespace:my-great-module' module and a corresponding meta.json file
+in the current directory.
+
+Next, update your meta.json and use 'viam module update' to push those changes to app.viam.com`,
+						Flags: []cli.Flag{
+							&cli.StringFlag{
+								Name:     "name",
+								Usage:    "name of your module (cannot be changed once set)",
+								Required: true,
+							},
+							&cli.StringFlag{
+								Name:  "public_namespace",
+								Usage: "the public namespace where the module will reside",
+							},
+							&cli.StringFlag{
+								Name:  "org_id",
+								Usage: "use namespace of given organization (alternative way of providing the public_namespace)",
```

Comment:
```suggestion
								Usage: "use the id of given organization (alternative way of providing the public_namespace)",
```

Reply:
Not super happy with my original or your edit. 

New suggestion: id of the organization that will host the module (alternative way of providing the public_namespace)

Reply:
SGTM!